doc/cwt-vc-verification-support.md

```
@@ -0,0 +1,119 @@
+```mermaid
+sequenceDiagram
+   Wallet->>CredentialsVerifier: verify credential
+   CredentialsVerifier->>CredentialVerifierFactory: Create verifier
+   CredentialVerifierFactory->>CwtVerifiableCredential: Create instance
+   CredentialsVerifier->>CwtVerifiableCredential: Validate
+   CwtVerifiableCredential->>CwtValidator: Validate
+   CwtValidator-->>CwtVerifiableCredential: Result
+   CwtVerifiableCredential-->>CredentialsVerifier: Result
+```
+
+###  Sequence diagram - validation process
```

markdownlint-cli2 (0.20.0)

[warning] Multiple spaces after hash on atx style heading

(MD019, no-multiple-space-atx)